src/collections/_documentation/error-reporting/configuration/index.md

```diff
@@ -54,7 +54,7 @@ Sets the environment.  This string is freeform and not set by default.  A releas
 with more than one environment to separate them in the UI (think `staging` vs `prod` or similar).
 
 By default the SDK will try to read this value from the `SENTRY_ENVIRONMENT`
-environment variable (except for the browser SDK where this is not applicable).
+environment variable (In the browser SDK, this will be read off of the `window` if available).
```

Comment:
Do you want to mention here that the reason it would be available is because of the webpack plugin?

Reply:
Oh this isn't even the right place to change this, but no, would rather keep webpack out of this, since there might be other reasons you would configure `window.SENTRY_RELEASE`.

Reply:
Fair. I'd still vote for telling people what the variable is called here, though. `(In the browser SDK, this will be read off of `window.SENTRY_RELEASE` if defined)` or some such.

Reply:
Yeah agreed, updated